cockroachcloud/cloud-sso.md

```diff
@@ -7,15 +7,15 @@ docs_area: manage
 
 Users may authenticate to the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) using Single Sign-On (SSO). GitHub, Google, and Microsoft are supported as identity providers (IdPs).
 
-[Enterprise authentication features](#private-preview-enterprise-authentication) are currently available in Private Preview. Reach out to your Cockroach Labs team to enquire about the private preview of Enterprise Authentication for Cloud SSO.
+[Enterprise authentication features](#preview-enterprise-authentication) are currently available in Preview. Reach out to your Cockroach Labs team to enquire about the preview of Enterprise Authentication for Cloud SSO.
```

Comment:
```suggestion
[Enterprise authentication features](#preview-enterprise-authentication) are available in Preview. Reach out to your Cockroach Labs team to enquire about the preview of Enterprise Authentication for Cloud SSO.
```

Reply:
Why is preview spelled with initial cap in first sentence and lc in second sentence?

Reply:
Replacing this with the include.